cmd/src/debug_comp.go

```diff
@@ -0,0 +1,255 @@
+package main
+
+import (
+	"archive/zip"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/sync/errgroup"
+
+	"golang.org/x/sync/semaphore"
+
+	"github.com/sourcegraph/src-cli/internal/cmderrors"
+)
+
+func init() {
+	usage := `
+'src debug comp' mocks docker cli commands to gather information about a docker-compose Sourcegraph instance.
```

Comment:
Doesn't actually mock anything, just invokes it.

Reply:
Corrected on all instances